airflow/jobs/scheduler_job_runner.py

```diff
@@ -774,9 +774,11 @@ def _process_executor_events(self, session: Session) -> int:
                     tags={"dag_id": ti.dag_id, "task_id": ti.task_id},
                 )
                 msg = (
-                    "Executor reports task instance %s finished (%s) although the "
-                    "task says it's %s. (Info: %s) Was the task killed externally?"
+                    f"The executor reported that the task instance {ti} finished with state {state}, but the task instance's state attribute is {ti.state}. "
```

Comment:
Perhaps we want to surround the state with `"..."` or <code>\`\`...\`\`</code>?

Reply:
Probably not particularly necessary since the state string is in a known pretty simple format. Up to the author I guess.